jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/URIUtil.java

```diff
@@ -1181,37 +1200,24 @@ else if (slash)
 
         // append any query
         if (i < end)
-            canonical.append(uri, i, end);
+            canonical.append(pathQuery, i, end);
 
         return canonical.toString();
     }
 
     /**
-     * @param path the encoded URI from the path onwards, which may contain query strings and/or fragments
-     * @return the canonical path, or null if path traversal above root.
-     * @deprecated Use {@link #canonicalURI(String)}
-     */
-    @Deprecated
-    public static String canonicalEncodedPath(String path)
-    {
-        return canonicalURI(path);
-    }
-
-    /**
-     * Convert a decoded URI path to a canonical form.
-     * <p>
-     * All segments of "." and ".." are factored out.
-     * Null is returned if the path tries to .. above its root.
+     * <p>Normalize a URI path by factoring out all segments of "." and "..".
+     * Null is returned if the path is normalized above its root.
      * </p>
      *
      * @param path the decoded URI path to convert. Any special characters (e.g. '?', "#") are assumed to be part of
      * the path segments.
      * @return the canonical path, or null if path traversal above root.
```

Comment:
Comment here still refers to it as the canonical path, not the normalized path.

Same for `normalizePathQuery` above.